js/values.js

```diff
@@ -48,21 +51,38 @@ function highlighting(user, baseUrl) {
       console.log("Injecting setup");
       if (!$(".side-panel").length) {
         $("body").append("<div id='add-highlight-button'><div id='add-symbol'>+</div></div>"
-          + "<div class='side-panel'><div class='annote-header'><img src='https://i.imgur.com/DxyYPfZ.png' class='pano-logo'><span class='pano'>PANO</span></div><div class='annote-text'></div></div>");
-      } 
+          + "<div class='side-panel'><div class='annote-header'><span class='pano'>Margins</span></div><div class='annote-text'></div></div>");
+      }
+
+      //save = new button
+      //comment bubble = mapping of the old b/c you want the side panel to open
 
       // $("body").append("<div id='side-panel-button'><img src='http://i.imgur.com/DxyYPfZ.png' class='pano-logo'></div>")
 
-      $('head').append("<script type='text/javascript' src='https://use.fontawesome.com/8c63cff961.js'>"
+      $("head").append("<script type='text/javascript' src='https://use.fontawesome.com/8c63cff961.js'>"
         + "<script src='https://code.jquery.com/ui/1.12.1/jquery-ui.js'></script>");
+        // + "<script src='https://code.jquery.com/jquery-1.8.0.min.js'></script>"
+        // + "<script>$(function() { $.scrollDepth();});</script>");
     
+
+        // + "<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery-scrolldepth/1.0.0/jquery.scrolldepth.js'></script>" 
+        //         + "<script>$(function() { $.scrollDepth();});</script>");
+
       if (highlighting_enabled) {
         reenable_highlighting();
       }
-
       // showAllComments();
     }();
 
+  //kept giving me scroll depth function not found
```

Comment:
do we need to keep this commented out code? (also scrolldepth might be a non-core jquery library that needs to be added before you can call it)

Reply:
I see it's included below. Hm I wonder if maybe it hasn't been imported yet in the code?

Reply:
ohh I see what happened, my latest push got added to the pull request. The scroll depth code is not ready for production. It's not connected to the server yet